SpatialGDK/Source/Public/Interop/SpatialReceiver.h

```diff
@@ -195,4 +204,7 @@ class USpatialReceiver : public UObject
 
 	TMap<Worker_RequestId, USpatialActorChannel*> PendingActorRequests;
 	FReliableRPCMap PendingReliableRPCs;
+
+	TMap<Worker_RequestId, EntityQueryDelegate> EntityQueryDelegates;
```

Comment:
This needs to be `Worker_RequestId_Key` to compile on linux

Reply:
Just ran a Linux build and it compiled fine. Are we sure we need this?

Reply:
Discussed offline ^^ is fine because is uint32_t